tsconfig.packages.json

```diff
@@ -13,7 +14,7 @@
      * `jest.config.packages.js`.
      */
     "paths": {
```

Comment:
i think you need to remove the comments here

Reply:
Yes I noticed this! I realized there are a bunch of comments in the tsconfig files from core. So, maybe I didn't need to update the `tsconfig.packages.build.json` to `jsonc`. We can probably revert that to stay aligned with core.